specs/agents/tracing-spans-drop-fast.md

```diff
@@ -0,0 +1,3 @@
+## Dropping fast spans
+
+TODO [`span_min_duration`](https://github.com/elastic/apm/pull/314) is viable again when tracking stats for fast spans [`tracing-spans-dropped-stats.md`](tracing-spans-dropped-stats.md).
```

Comment:
I think we should use `span_min_duration` as the second to last line of defense (before the hard limit).